types/makeup-screenreader-trap/makeup-screenreader-trap-tests.ts

```diff
@@ -0,0 +1,7 @@
+import { trap, untrap } from 'makeup-screenreader-trap';
+
+const widgetEl: HTMLElement | null = document.querySelector('.expander');
+
+trap(widgetEl);
```

Comment:
It doesn't look like the library actually accepts `null`.

Reply:
Fixed.